graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/model/GraphQLResponse.kt

```diff
@@ -29,6 +31,12 @@ data class GraphQLResponse(
 )
 
 fun ExecutionResult.toGraphQLResponse(): GraphQLResponse {
+    errors?.forEach {
```

Comment:
While this indeed fixes the issue, it is very specific to that single case and if another GraphQL error is returned that is also an Exception we will be returning stacktraces as well.

Let's change it to more generic
```
val filteredErrors = if (errors?.isNotEmpty() == true) {
    errors.map {
        when (it) {
            is SimpleKotlinGraphQLError -> it
            is Exception -> SimpleKotlinGraphQLError(exception = it, locations = it.locations, path = it.path, errorType = it.errorType)
            else -> it
        }
    }
} else {
    null
}
```

Reply:
+1, let's make this more generic

Reply:
Good call. Made the change. @dariuszkuc @smyrick